static/src/js/App.jsx

```diff
@@ -48,8 +54,54 @@ export const App = () => {
     document.body.classList.remove('is-loading')
   }, [])
 
+  const { addNotification } = useNotificationsContext()
+
+  const {
+    user, setProviderId, setProviderIds
+  } = useAppContext()
+
+  const [getProviders] = useLazyQuery(GET_ACLS, {
+    variables: {
+      params: {
+        includeFullAcl: true,
+        pageNum: 1,
+        pageSize: 2000,
+        permittedUser: 'typical',
```

Comment:
This needs the users actual id

Reply:
From PR's review: this fix is in another ticket later

Reply:
This change still needs to happen. `typical` will not work in SIT.